pubspec.yaml

```diff
@@ -2,28 +2,54 @@ name: flutter_ecommerce
 description: A new Flutter project.
 
 publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
+  sdk: ">=2.16.1 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
+
```

Comment:
Try to remove the white spaces between the dependencies
Just one line between the different headlines and between the dependencies themselves: no space.